src/context/SnackbarContext.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC, ReactNode, createContext, useState } from 'react'
+import CustomSnackbar from '../components/CustomSnackbar/CustomSnackbar'
+
+type SnackbarData = {
+  open: boolean
+  error: boolean
+  message?: string
+}
+
+type SnackbarContext = {
+  snackbarData: SnackbarData
+  showSnackbar: (data: Omit<SnackbarData, 'open'>) => void
+  hideSnackbar: () => void
+}
+
+export const SnackbarContext = createContext<SnackbarContext | undefined>(
+  undefined
+)
```

Comment:
you can do just this
```suggestion
export const SnackbarContext = createContext<SnackbarContext>(
 {} as SnackbarContext
)
```

Reply:
done